src/ClassicUO.Client/LegionScripting/docs/API.py

```
@@ -1464,6 +1479,27 @@ def MoveItemOffset(serial: "int", amt: "int" = 0, x: "int" = 0, y: "int" = 0, z:
     """
     pass
 
+def PickUpToCursor(serial: "int" = 0, amt: "int" = 0) -> None:
+    """
+     Picks up an item from the game world and places it onto the mouse cursor.
+    
+    """
+    pass
+
+def DropFromCursor(serial: "int" = 0, x: "int" = 1337, y: "int" = 1337, z: "int" = sbyte.MaxValue, container: "int" = 1337) -> None:
```

Ruff (0.15.1)

[error] Undefined name `sbyte`

(F821)